pkg/testing/ethmultisig_test.go

```diff
@@ -97,7 +108,46 @@ func (suite *ETHMultisigTestSuite) TestMultisig() {
 		ok, err := suite.chain.multisigClient.VerifyClientConsensusState(
 			suite.chain.CallOpts(ctx, 0),
 			suite.chain.ContractConfig.GetIBCHostAddress(),
-			clientID, 1, counterpartyClientID, consensusHeight.RevisionHeight, prefix, proofBytes, anyConsensusStateBytes,
+			clientID,
+			multisigclient.HeightData{
+				RevisionNumber: 0,
+				RevisionHeight: 1,
+			},
+			counterpartyClientID,
+			multisigclient.HeightData{
+				RevisionNumber: consensusHeight.RevisionNumber,
+				RevisionHeight: consensusHeight.RevisionHeight,
+			},
+			prefix, proofBytes, anyConsensusStateBytes,
+		)
+		suite.Require().NoError(err)
+		suite.Require().True(ok)
+	}
+
+	// VerifyClientConsensusState consensusHeight.revisionNumber=1
+	{
+		targetConsensusState := makeMultisigConsensusState(nil, "tester2", uint64(time.Now().UnixNano()))
+		consensusHeight := clienttypes.NewHeight(1, 100)
+		proofConsensus, _, err := prover.SignConsensusState(proofHeight, counterpartyClientID, consensusHeight, targetConsensusState)
+		suite.Require().NoError(err)
+		anyConsensusStateBytes, err := suite.cdc.MarshalInterface(targetConsensusState)
+		suite.Require().NoError(err)
+		proofBytes, err := proto.Marshal(proofConsensus)
+		suite.Require().NoError(err)
+		ok, err := suite.chain.multisigClient.VerifyClientConsensusState(
+			suite.chain.CallOpts(ctx, 0),
+			suite.chain.ContractConfig.GetIBCHostAddress(),
+			clientID,
+			multisigclient.HeightData{
+				RevisionNumber: 0,
```

Comment:
Is this `0` correct? (If incorrect, why does it pass the test?)

Reply:
This line is correct because it corresponds to the following
https://github.com/datachainlab/ibc-ethmultisig-client/blob/bump-yui-ibc-solidity/pkg/testing/ethmultisig_test.go#L52